src/image/image.css

```diff
@@ -1,178 +1,178 @@
 .img.webpack {
-  background-image: url(~devtools/client/debugger/images/sources/webpack.svg);
+  background-image: url(~devtools/client/debugger/images/sources/webpack.svg) !important;
```

Comment:
as a reference, why do we need this?

Reply:
There's an import order problem where (some, at least) of these weren't taking effect because they used to be. imported after another rule that had the same precedence. This was my "let's just ship this already" bail out for that problem. Follow up required.